src/checkConsent.js

```diff
@@ -0,0 +1,10 @@
+const LICENSES = (process.env.LICENSE_URL || '').split(',');
```

Comment:
Do you need to trim space first before splitting to accommodate someone pass params with space after comma.

Reply:
Good point. I trimmed each URL after splitting, because I think it's possible for developers to write something like `URL1, URL2`.